src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/PackageSearchCommand.cs

```diff
@@ -66,6 +66,24 @@ public static void Register(CliCommand rootCommand, Func<ILoggerWithColor> getLo
                 Arity = ArgumentArity.ExactlyOne
             };
 
+            var format = new CliOption<string>("--format")
+            {
+                Description = Strings.pkgSearch_FormatDescription,
+                Arity = ArgumentArity.ExactlyOne
+            };
+
+            var verbosity = new CliOption<string>("--verbosity")
+            {
+                Description = Strings.pkgSearch_VerbosityDescription,
```

Comment:
How about leveraging existing string resource that other commands use for verbosity? for example https://github.com/NuGet/NuGet.Client/blob/4ea7c85930a70c19f69293b572136e46040ff745/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/VerifyCommand.cs#L48

Reply:
I could not use it for this command because there is a difference between the options the two messages provide. For `Verbosity_Description`  Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic]